core/src/memory.rs

```diff
@@ -36,10 +38,23 @@ const GEAR_PAGE_SIZE: usize = 0x1000;
 /// Number of gear pages in one wasm page
 const GEAR_PAGES_IN_ONE_WASM: u32 = (WASM_PAGE_SIZE / GEAR_PAGE_SIZE) as u32;
 
-pub use gear_core_errors::MemoryError as Error;
+/// Buffer for gear page data.
+pub type PageBuf = Box<[u8; GEAR_PAGE_SIZE]>;
```

Comment:
What about to make `PageBuf` as newtype and move functions below to its impl?

Reply:
Yes, it's good idea. I will make it in next PR if this will be committed

Reply:
Ok done now